### PR TITLE
added prettier config, components, and sass

### PR DESCRIPTION
**Pull request by m-spilman**

### CODE REVIEW

**Review by ahaly92 — APPROVED** [review]

**Review by BarretteBanner — APPROVED** [review]

**Comment by m-spilman** [comment]
changed <a> to <div> cleaned up some names (camelCase). no more errors in console : ) 
let me know if you wanted this on a new branch - so further updates will be on new branches. And if once each pull gets approved if you want us squash? - Thanks for your help!

**Review by BarretteBanner — APPROVED** [review]
Looks great! This can be on this branch. It's once you merge that you should create a new branch. And you should always rebase / squash before a pr (and after updating a pr)  that way when you're told to ship it you can just press 'squash and merge'

File: src/components/Header/Header.js
Comment (ahaly92) [inline src/components/Header/Header.js]: use react's Link component here instead of a div with an href.

File: src/components/Main/Main.js
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's have your paths look like this: "/admin-orders"

File: src/components/Main/Main.js
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's try adding a path that re-directs us to '/' if they type in something that doesn't match any of the options in the url. There are multiple ways to do this so when you find a solution you like reach out so we can discuss first before you implement it.

File: src/components/Main/Main.js
Comment (BarretteBanner) [inline src/components/Main/Main.js]: Let's make this one re-direct to the '/' and maybe change the '/' to '*'

**Review by BarretteBanner — APPROVED** [review]
Well done!

**Pull request merged**
